third_party/lua-yaml/lyaml.cc

```diff
@@ -618,7 +618,9 @@ static int dump_node(struct lua_yaml_dumper *dumper)
    yaml_event_t ev;
    yaml_scalar_style_t style = YAML_PLAIN_SCALAR_STYLE;
    int is_binary = 0;
-   char buf[MAX(FPCONV_G_FMT_BUFSIZE, DT_TO_STRING_BUFSIZE)];
+   char buf[DT_IVAL_TO_STRING_BUFSIZE];
+   static_assert(DT_IVAL_TO_STRING_BUFSIZE > FPCONV_G_FMT_BUFSIZE);
+   static_assert(DT_IVAL_TO_STRING_BUFSIZE > DT_TO_STRING_BUFSIZE);
```

Comment:
Static assert needs a message. Otherwise compilation works not everywhere.

Reply:
Fixed.